src/packages/cli/src/Init.ts

```diff
@@ -82,6 +82,12 @@ export const defaultURL = (
   }
 }
 
+export const defaultGitIgnore = () => {
+  return `node_modules
+# Keep environment variables out of version control
+.env`
```

Comment:
```suggestion
.env
`
```

To have an empty line return at end of file.